packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -446,6 +455,19 @@ export class SingleAddressWallet implements ObservableWallet {
     this.#logger.debug('Shutdown');
   }
 
+  async #getSignatures(txInternals: TxInternals, extraSigners?: TransactionSigner[]) {
+    const signatures: Cardano.Signatures = await this.keyAgent.signTransaction(txInternals);
```

Comment:
```suggestion
    const signatures: Cardano.Signatures = await this.keyAgent.signTransaction(txInternals, signingOptions);
```

Reply:
Addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/404/commits/5a045e1d9b2a4c6faf5dbb042f038da9fb2730a7 (and was properly linted here https://github.com/input-output-hk/cardano-js-sdk/pull/404/commits/480ab9087c32cf72254f2609d04d91aa49d5c3e6)